luxonis_ml/nn_archive/config_building_blocks/base_models/head.py

```diff
@@ -144,6 +163,15 @@ def validate_label_type(
             raise ValueError("Invalid family")
         return value
 
+    @field_validator("metadata")
+    def validate_metadata(cls, value):
+        if not isinstance(value, HeadSegmentationMetadata):
```

Comment:
Can this ever happen with the `pydantic` validation already in place?

Reply:
Thanks, removed in [1031e73](https://github.com/luxonis/luxonis-ml/pull/127/commits/1031e733652d54e98130f6baf7400c621414b5eb).